frontend/packages/dev-console/src/components/edit-deployment/EditDeployment.tsx

```diff
@@ -61,24 +62,36 @@ const EditDeployment: React.FC<EditDeploymentProps> = ({ heading, resource, name
       deploymentRes = convertEditFormToDeployment(values.formData, resource);
     }
 
-    const resourceCall = k8sUpdate(
-      resourceType === Resources.OpenShift ? DeploymentConfigModel : DeploymentModel,
-      deploymentRes,
-      namespace,
-      name,
-    );
+    const resourceCall = isNew
+      ? k8sCreate(DeploymentModel, deploymentRes)
```

Comment:
can you use `K8sCreateResource`  and `k8sUpdateResource` for update ?

Reply:
Yes, have updated the code